Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -79,8 +93,77 @@ private void ToggleConsole()
         }
     }
 
+    #region FreelookCam
+    /// <summary>
+    /// The player unposseses the character
+    /// and become a free look cam that can
+    /// fly arround
+    /// </summary>
+    private void ToggleFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return;
+
+        if (!_isInFreeLookCamMode)
+        {
+             _spawnedFreeLookCam = EnterFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Exit Free Look Cam";
+        }
+        else if (_isInFreeLookCamMode)
+        {
+            ExitFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Enter Free Look Cam";
+        }
+
+    }
+    /// <summary>
+    /// Puts the player in free look cam
+    /// mode
+    /// </summary>
+    private GameObject EnterFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return null;
+
+        CinemachineVirtualCamera _playerVirtualCam = GameObject.FindObjectOfType<CinemachineVirtualCamera>();
+
+        //spawn free look cam
+        GameObject _tempFreeLookCam = Instantiate(_freeLookCam,
+            GameObject.FindGameObjectWithTag("MainCamera").transform.position, Quaternion.identity,
+            GameObject.FindObjectOfType<BoatScriptTag>().transform);
```

Comment:
We could probably just use the instance of the BoatMover instead

Reply:
^